lib/styles/settings/_media-queries.scss

```diff
@@ -9,3 +9,7 @@
 @function breakpoint-l() {
 	@return '(min-width: 1280px)';
 }
+
+@function breakpoint-xl() {
```

Comment:
should we show it in https://github.com/bethinkpl/design-system/blob/442a99fffbc3cbb659d57288e1c44e9407c9f40b/.storybook/preview.js#L33 ?